manifests/params.pp

```diff
@@ -61,6 +63,27 @@
       $repo_location        = 'https://repos.influxdata.com/'
       $service_enable       = true
       $service_ensure       = running
+      $service_flags        = undef
+      $service_hasstatus    = true
+      $service_restart      = 'pkill -HUP telegraf'
+    }
+    'FreeBSD' : {
```

Comment:
The [metadata](https://github.com/voxpupuli/puppet-telegraf/blob/master/metadata.json) should include FreeBSD support at specific versions.

Reply:
Also README should be updated to list FreeBSD.

Reply:
Metadata and README fixed in 0391569e4c441e219c83a01f595e28992c6d3533

Joris